### DISCUSSION TITLE
NiceGUI pages will automatically refresh if not operated for a long time.

### DISCUSSION THREAD

**Discussion by HycJack** [zauberzeug/nicegui#3373] [Q&A]
### Question

When I open the browser, visit the NiceGUI page, perform some operations, and then check the NiceGUI page the next day, it will automatically refresh. Can the refreshing be prevented?  Does this relate to the implementation principles of NiceGUI?

**Comment by MohAid** [comment]
For more details about this issue you can read here #1100, and as a solution you may want to use `app.storage.tab` see [this replay](https://github.com/zauberzeug/nicegui/discussions/1100#discussioncomment-9033154). and also take this #3312 into account.
